frontend/src/components/modals/ProfileDialog.tsx

```diff
@@ -0,0 +1,162 @@
+                dispatch(setAppState(GlobalState.LOADING));
+                dispatch(resetMaps());
+                setAsk(false);
+            });
+        };
+        setAction({act: vl});
+        setAsk(true);
+    }
+
+    return (
+        <>
+        <YouSureDialog open={ask} close={close} accept={async () => {
+            await action.act();
+            close();
+        }}/>
+        <Dialog
+            open={open}
+            aria-labelledby="profile-dialog-title"
+            aria-describedby="profile-dialog-description">
+            <DialogTitle sx={{ m: 0, p: 2, width: "600px"}}>
+                Настройки профиля
+                <IconButton
+                    aria-label="close"
+                    onClick={onClose}
+                    sx={{
+                        position: 'absolute',
+                        right: 8,
+                        top: 8
+                    }}
+                >
+                    <Close />
+                </IconButton>
+            </DialogTitle>
+            <DialogContent dividers sx={{width: "100%"}}>
+                <Grid container spacing={2}>
+                    <Grid item xs={12}>
+                        <TextField fullWidth
+                                   type="password"
+                                   value={pass}
+                                   label="Пароль"
+                                   onChange={(e) => setPass(e.target.value)}
+                                   error={passError}
+                                   />
+                    </Grid>
+                    <Divider/>
+                    <Grid item xs={12}>
+                        <Typography variant="h5" color="error">
+                            Смена пароля
+                        </Typography>
+                    </Grid>
+                    <Grid item xs={8}>
+
+                        <TextField fullWidth
+                                   type="password"
+                                   value={newPass}
+                                   disabled={passError}
+                                   label="Новый пароль"
+                                   onChange={(e) => setNewPass(e.target.value)}
+                                   error={pass == user.password}
+                        />
+                    </Grid>
+                    <Grid item xs={4}>
+                        <Button onClick={chn}>
+                            Сменить
+                        </Button>
+                    </Grid>
+
+                    <Grid item xs={12}>
+                        <Typography variant="h5" color="error">
+                            Удаление профиля
+                        </Typography>
+                    </Grid>
+                    <Grid item xs={8}>
+                        <FormControlLabel control={<Switch checked={swt} onChange={(e)=>setSwt(e.target.checked)} defaultChecked />} label="Я знаю, что удаление уничтожит все созданные расписания." />
```

Comment:
Для таких больших компонентов, которые даже не влезают в строчку, да и вообще, желательно ко всем компонентов, где значений больше одного лучше использовать такое форматирование
```suggestion
<FormControlLabel
    label="Я знаю, что удаление уничтожит все созданные расписания."
    control={
        <Switch
            checked={swt}
            onChange={(e)=>setSwt(e.target.checked)}
            defaultChecked
        />
    }
/>
```
То есть каждый параметр в отдельной строчке
Здесь и в остальных местах

Reply:
И слово "уничтожит", хорошо бы заменить)